datalad_neuroimaging/tests/utils.py

```diff
@@ -32,6 +32,12 @@ def get_dicom_dataset(flavor):
         dataset=srcrepo,
         path=opj(srcrepo.path, 'datalad_neuroimaging', 'tests', 'data',
                  'dicoms', flavor))
+    # Note: This is broken with datalad 0.12.0rc6; install returns None
+    # see datalad gh-3836
+    # workaround it:
+    ds = Dataset(opj(srcrepo.path, 'datalad_neuroimaging', 'tests', 'data',
+                 'dicoms', flavor))
+    assert ds.is_installed()
```

Comment:
I could be wrong, but I feel that it might be quite a common pattern which would keep coming up, and better be resolved in datalad itself instead of breeding workarounds.  If anything I would have at least absorbed common "logic" for path into `dspath = opj(...)` and use dspath in both lines

Reply:
Agree - it should be actually solved in datalad. However, I just need to see, whether I found all the spots, where datalad-hirni and its dependencies fail to properly work with rc6.